java/runtime/src/main/java/org/ray/runtime/functionmanager/PyFunctionDescriptor.java

```diff
@@ -0,0 +1,25 @@
+package org.ray.runtime.functionmanager;
+
+/**
+ * Represents metadata of a Python function.
+ */
+public class PyFunctionDescriptor {
```

Comment:
For the sake of a little more type safety, how about having both this and FunctionDescriptor extend the same interface or base class?

Reply:
Thanks, will change.